src/Core/Repositories/IOrganizationUserRepository.cs

```diff
@@ -37,5 +37,6 @@ public interface IOrganizationUserRepository : IRepository<OrganizationUser, Gui
         Task DeleteManyAsync(IEnumerable<Guid> userIds);
         Task<OrganizationUser> GetByOrganizationEmailAsync(Guid organizationId, string email);
         Task<IEnumerable<OrganizationUserPublicKey>> GetManyPublicKeysByOrganizationUserAsync(Guid organizationId, IEnumerable<Guid> Ids);
+        Task<IEnumerable<OrganizationUserUserDetails>> GetManyByRoleAsync(Guid organizationId, OrganizationUserType role);
```

Comment:
Please rename this to better indicate it's function/intent.

```suggestion
        Task<IEnumerable<OrganizationUserUserDetails>> GetManyByMinimumRoleAsync(Guid organizationId, OrganizationUserType minRole);
```

Reply:
Changed